src/lib/storage/shadowed_vol_list.rb

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2016 Novell, Inc.
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE.
+#
+# To contact SUSE about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+Yast.import "FileSystems"
+
+module Yast
+  class ShadowedVolList
+    include Enumerable
+    include Yast::Logger
+
+    attr_reader :partition, :mntpoint
+
+    def initialize(partition: nil, mntpoint: nil)
```

Comment:
NP: mntpoint is not really that intuitive for a named parameter. Any good reason against mount_point which would be better to memorize?